test/valid-packages-test.js

```diff
@@ -306,6 +307,24 @@ packages.map(function(pkg) {
                 pkgName(pkg) + ": Need to remove \"/\" at the front or the last of basePath in package.json");
         }
       }
+  }
+  packageVows[pname + ": There can't be duplicate keywords"] = function(pkg) {
+      var json = parse(pkg, true);
+      var keywords = json.keywords;
+      var dul_key = [];
+      if (keywords !== undefined) {
+        keywords.sort();
+        dul_key = keywords.filter (function (dul, i, temp) {
+          if (i >= 0 && dul === temp[i-1] && dul !== temp[i-2]) {
+            return dul;
+          }
+        });
+        dul_key = _.map (dul_key, function(item) {
+          return " " + item;
+        });
+      }
+      assert.ok(dul_key.length === 0,
+        pkgName(pkg) + ": Need to remove duplicate keywords: " + dul_key);
```

Comment:
@kennynaoh I think you don't need space after colon mark here because elements with leading space in the array.